.github/workflows/unix.yml

```
@@ -78,6 +78,30 @@ jobs:
         export OPENPMD_BP_BACKEND=ADIOS1
         ctest --output-on-failure
 
+  clang5_nopy_ompi_h5_ad2_newLayout_bp3:
```

copy-paste typo:
```suggestion
  clang5_nopy_ompi_h5_ad2_newLayout:
```

the `_bp3` indicated that we are using the BP3 engine in a test with ADIOS2

---

done in 23fbf0efcd8d544a82853a9242f5c2d345ddd6f8